sdk/servicebus/azure-servicebus/azure/servicebus/aio/management/_utils.py

```diff
@@ -89,6 +89,26 @@ async def extract_data_template(feed_class, convert, feed_element):
     return next_link, iter(list_of_qd)  # when next_page is None, AsyncPagedItem will stop fetch next page data.
 
 
+async def extract_rule_data_template(feed_class, convert, feed_element):
+    """Special version of function extrat_data_template for Rule.
+
+    Pass both the XML entry element and the rule instance to function `convert`. Rule needs to extract
+    KeyValue from XML Element and set to Rule model instance manually. The msrest serialization/deserialization
```

Comment:
is this something msrest should handle? (in the sense of should we be filing an issue against msrest?) or is it very bespoke and not generalizable.

Reply:
I requested a feature for autorest to work both with an xml tag that has both attributes and text. See [here](https://github.com/Azure/autorest/issues/3535). This is the general part of this problem. And this will greatly help us.
autorest is making progress but doesn't have a commitment when to release it so I wrote this workaround by manipulating xml Element. 
Even after autorest/msrest enhancement is released, the "type" of the "Value" below is not generalized enough for msrest to handle it. It will still help us because we won't need to manipulate xml Element.
```xml
<Value xmlns:d6p1="http://www.w3.org/2001/XMLSchema-instance" xsi:type="d6p1:string">1</Value>
```